delta/middleman.py

```diff
@@ -57,7 +69,8 @@ def forward(Q, cfg, timeout):
             break
 
         writer.BeginStep()
-        writer.put_data(msg.data)
+        # (2020/11) jyc: This is a hack. Need to wrap as ecei_chunk object
+        writer.put_data_np(msg.data, {"tidx": msg.tstep_idx})
```

Comment:
You can pass msg directly to writer.put_data. The data_class assumes that the data member is called .data, so any class that has a .data member should be fine as an argument.

Reply:
I see. I just updated the code.